test/test_unit_keystore_credentials_provider.py

```diff
@@ -30,16 +35,66 @@ def test_load_credentials_do_not_exist_raises_credential_retrieval_exception(sel
             credential_provider = KeystoreCredentialsProvider()
             credential_provider.load('https://someurl')
 
+    @patch.object(KeystoreCredentialsProvider, 'is_exists', return_value=True)
+    @patch.object(KeystoreAdapter, "get_password", return_value="password")
+    def test_load_credentials_calls_get_password(self, mock_store_adapter, mock_cred_provider):
+        credential_provider = KeystoreCredentialsProvider()
+        credentials_data = credential_provider.load('https://someurl')
+        calls = [call('https://someurl', 'machine'), call('https://someurl', 'login'),
+                 call('https://someurl', 'password')]
+        mock_store_adapter.assert_has_calls(calls)
+        self.assertEquals("password", credentials_data.machine)
+        self.assertEquals("password", credentials_data.login)
+        self.assertEquals("password", credentials_data.password)
+
     @patch.object(KeystoreAdapter, 'set_password')
     def test_update_api_key_calls_methods_properly(self, mock_store_adapter):
         # mock_store_adapter.set_password.return_value=None
         credential_provider = KeystoreCredentialsProvider()
         credential_provider.update_api_key_entry('someusertoupdate', MockCredentials, 'newapikey')
-        calls = [call('https://somemachine', 'machine', 'https://somemachine'), call('https://somemachine', 'login', 'someusertoupdate'), call('https://somemachine', 'password', 'newapikey') ]
+        calls = [call('https://somemachine', 'machine', 'https://somemachine'),
```

Comment:
I would use constants here because if 'machine', 'login' , etc changes one day, the test will break :l see constants.py

Reply:
I think I wrote this so I am giving a critique on myself :P

Reply:
Done